### PR TITLE
Challeng 04, 05, and 06 - Flexbox party

### PR DESCRIPTION
**Pull request by yadleo**

### CODE REVIEW

File: daniel/challenge04/style.css [hunk 1]
```diff
@@ -19,65 +19,101 @@
...
+  font-family: 'Roboto', sans-serif;
+  font-size: 1.3rem;
 }
 
 h1 {
-    font-size: 3rem;
+  font-size: 3rem;
+}
+
+.c-fr-lilac {
```
Comment (benbli) [inline daniel/challenge04/style.css:47]: what does `fr` stand for?

**Review by benbli — APPROVED** [review]

File: daniel/challenge04/style.css [hunk 2]
```diff
@@ -19,65 +19,101 @@
...
+
+.pb-100px {
+  padding-bottom: 100px;
+}
+
+.text-align-center {
+  text-align: center;
+}
+
+.line-height-1-5rem {
```
Comment (benbli) [inline daniel/challenge04/style.css:63]: I think the `1-5rem` notation is a bit awkward, but I don't have a better notation that I usually use. I'm curious how tailwind does it or other css util libs

File: daniel/challenge04/style.css (hunk 1 above)
Comment (yadleo) [reply daniel/challenge04/style.css:47]: yeah... i used this site for some of these names
https://www.color-name.com/

File: daniel/challenge04/style.css (hunk 2 above)
Comment (yadleo) [reply daniel/challenge04/style.css:63]: yeah. I felt awkward when I was naming it, but couldn't really find anything aside from naming like the following

- `.line-height-sm`
- `.line-height-md`
- `.line-height-lg`

**Pull request merged**